### PR TITLE
Fix potential NPE

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: spring-cloud-kubernetes-discovery/src/main/java/org/springframework/cloud/kubernetes/discovery/KubernetesDiscoveryClient.java [hunk 1]
```diff
@@ -16,11 +16,7 @@
  */
 package org.springframework.cloud.kubernetes.discovery;
 
-import io.fabric8.kubernetes.api.model.EndpointAddress;
-import io.fabric8.kubernetes.api.model.EndpointPort;
-import io.fabric8.kubernetes.api.model.EndpointSubset;
-import io.fabric8.kubernetes.api.model.Endpoints;
-import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.api.model.*;
```
Comment (ryanjbaxter) [inline spring-cloud-kubernetes-discovery/src/main/java/org/springframework/cloud/kubernetes/discovery/KubernetesDiscoveryClient.java]: `*` imports

Comment (geoand) [reply spring-cloud-kubernetes-discovery/src/main/java/org/springframework/cloud/kubernetes/discovery/KubernetesDiscoveryClient.java]: I really need to setup my IDE to not do that :P

Comment (geoand) [reply spring-cloud-kubernetes-discovery/src/main/java/org/springframework/cloud/kubernetes/discovery/KubernetesDiscoveryClient.java]: Fixed

**Comment by geoand** [comment]
@ryanjbaxter Is this good to merge?

**Pull request merged**